i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/infrastructure-active-directory-nrpe.md

```diff
@@ -1,84 +1,285 @@
+> Pour obtenir ce nouveau format de métrique, incluez la valeur **--use-new-perfdata** dans la macro de service **EXTRAOPTIONS**.
+
+</TabItem>
+</Tabs>
 
-## Prerequisites
+## Prérequis
 
 ### Centreon NSClient++
 
-To monitor an *Active Directory* domain controller through NRPE, install the Centreon packaged version 
-of the NSClient++ agent. Please follow our [official documentation](../getting-started/how-to-guides/centreon-nsclient-tutorial.md) 
-and make sure that the **NRPE Server** configuration is correct.
+Pour surveiller les ressources *Windows* via NRPE, installez la version Centreon
+de l'agent NSClient++. Veuillez suivre notre [documentation officielle](../getting-started/how-to-guides/centreon-nsclient-tutorial.md) et assurez-vous que la configuration du **serveur NRPE** est correcte.
 
-## Installation 
+## Installer le connecteur de supervision
+
+### Pack
+
+1. Si la plateforme est configurée avec une licence *online*, l'installation d'un paquet
+n'est pas requise pour voir apparaître le connecteur dans le menu **Configuration > Gestionnaire de connecteurs de supervision**.
+Au contraire, si la plateforme utilise une licence *offline*, installez le paquet
+sur le **serveur central** via la commande correspondant au gestionnaire de paquets
+associé à sa distribution :
 
 <Tabs groupId="sync">
-<TabItem value="Online License" label="Online License">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
+
+```bash
+dnf install centreon-pack-infrastructure-active-directory-nrpe
+```
 
-1. Install the Centreon NRPE Client package on every Poller expected to monitor *Active Directory*:
+</TabItem>
+<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">
 
 ```bash
-yum install centreon-nrpe-plugin
+dnf install centreon-pack-infrastructure-active-directory-nrpe
 ```
 
-2. On the Centreon Web interface, install the Centreon Pack *Active Directory* 
-from the **Configuration > Monitoring Connector Manager** page
+</TabItem>
+<TabItem value="Debian 11" label="Debian 11">
+
+```bash
+apt install centreon-pack-infrastructure-active-directory-nrpe
+```
 
 </TabItem>
-<TabItem value="Offline License" label="Offline License">
+<TabItem value="CentOS 7" label="CentOS 7">
+
+```bash
+yum install centreon-pack-infrastructure-active-directory-nrpe
+```
+
+</TabItem>
+</Tabs>
+
+2. Quel que soit le type de la licence (*online* ou *offline*), installez le connecteur **Active Directory**
+depuis l'interface web et le menu **Configuration > Gestionnaire de connecteurs de supervision**.
+
+### Plugin
 
-1. Install the Centreon package on every Poller expected to monitor *Active Directory*:
+Utilisez les commandes ci-dessous en fonction du gestionnaire de paquets de votre système d'exploitation :
+
+<Tabs groupId="sync">
+<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">
 
 ```bash
-yum install centreon-nrpe-plugin
+dnf install nagios-plugins-nrpe
```

Comment:
last version had centreon-nrpe-plugin, is it normal that we have nagios now?

Reply:
yes